app/components/dashboard/audit_information_component.html.erb

```diff
@@ -46,16 +45,16 @@ TBD:  make the audit check description into a tooltip?
       <td>checksum_validation</td>
       <td><strong>Weekly</strong>: 1am on Sunday<br/><em>for CompleteMoabs with expired checks</em></td>
       <td><strong>subset</strong>:<br/>expired checksums queued by associated MoabStorageRoot</td>
-      <td class="text-end<%= ' table-danger' if moab_audits_older_than_threshold? %>">Moab audits older than <%= moab_audit_age_threshold %>:<hr/><%= num_moab_audits_older_than_threshold %></td>
+      <td class="text-end<%= ' table-warning' if moab_audits_older_than_threshold? %>">Moab audits older than <%= moab_audit_age_threshold %>:<hr/><%= num_moab_audits_older_than_threshold %></td>
       <td class="text-end<%= ' table-danger' if CompleteMoab.invalid_checksum.count != 0 %>"><%= CompleteMoab.invalid_checksum.count %></td>
-      <td class="text-center"><%= num_expired_checksum_validation %><br/><br/>(passing checks expire 90 days after they are run)</td>
+      <td class="text-center<%= ' table-warning' if num_expired_checksum_validation != 0 %>"><%= num_expired_checksum_validation %><br/><br/>(passing checks expire 90 days after they are run)</td>
```

Comment:
I'd encourage moving the logic into the ruby part of the component here and elsewhere.